### PR TITLE
HPCC-13527 Add throttling settings to ConfigMgr

### PR DESCRIPTION
**Pull request by garonsky**
Signed-off-by: Gleb Aronsky gleb.aronsky@lexisnexis.com

### CODE REVIEW

**Comment by garonsky** [comment]
@jakesmith please review

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success
HPCC Start: OK
HPCC Stop: OK

File: initfiles/componentfiles/configxml/dafilesrv.xsd [hunk 1]
```diff
@@ -93,6 +114,27 @@
...
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>Defines the maximum number of concurrent dafilesrv requests allowed. Requests that exceed the limit will be delayed. A value of 0 disables throttling. This a global setting.</tooltip>
+                    </xs:appinfo>
+                </xs:annotation>
+            </xs:attribute>
+            <xs:attribute name="throttleDelayMs" type="xs:nonNegativeInteger" use="optional" default="5000">
+                <xs:annotation>
+                    <xs:appinfo>
+                        <tooltip>Defines how many millisconds delays requests will be delayed by. This a global setting.</tooltip>
```
Comment (jakesmith) [inline initfiles/componentfiles/configxml/dafilesrv.xsd]: >Defines how many millisconds delays requests will be delayed by

Defines how many **milliseconds** **delayed** requests will be delayed by

File: initfiles/componentfiles/configxml/dafilesrv.xsd [hunk 2]
```diff
@@ -54,6 +54,27 @@
...
+                            <xs:annotation>
+                                <xs:appinfo>
+                                    <tooltip>Defines the maximum number of concurrent dafilesrv requests allowed. Requests that exceed the limit will be delayed. A value of 0 disables throttling. Overrides global settting.</tooltip>
+                                </xs:appinfo>
+                            </xs:annotation>
+                        </xs:attribute>
+                        <xs:attribute name="throttleDelayMs" type="xs:nonNegativeInteger" use="optional" default="5000">
+                            <xs:annotation>
+                                <xs:appinfo>
+                                    <tooltip>Defines how many millisconds delays requests will be delayed by. Overrides global settting.</tooltip>
```
Comment (jakesmith) [inline initfiles/componentfiles/configxml/dafilesrv.xsd]: > Defines how many millisconds delays requests will be delayed by

Defines how many **milliseconds** **delayed** requests will be delayed by

**Comment by jakesmith** [comment]
@garonsky - looks good, minor comments re. typo's.

**Comment by garonsky** [comment]
Minor changes to tooltip/docs have been made.

@richardkchapman this is ready to merge.

**Pull request merged**